### ISSUE TITLE
2.0.0-dev is not getting accepted as valid version

### ISSUE THREAD

**Issue by avsej** [kubernetes/minikube#8560]
```
$ minikube start --driver=podman --container-runtime=cri-o
😄  minikube v1.12.0-beta.0 on Fedora rawhide
✨  Using the podman (experimental) driver based on user configuration

❗  'podman' driver reported an issue: Invalid character(s) found in prerelease "dev\n"
💡  Suggestion: Cant verify minimum required version for podman . See podman website for installation guide.
📘  Documentation: https://podman.io/getting-started/installation.html

💣  Failed to validate 'podman' driver
```

```
$ podman version
Version:      2.0.0-dev
API Version:  1
Go Version:   go1.14.3
Git Commit:   9980991f9be8f5fbb4e87caf5ac34ce2575a232c
Built:        Thu Jun 18 03:00:00 2020
OS/Arch:      linux/amd64
```

https://github.com/kubernetes/minikube/blob/72e13c1eb87fa29461cd62178c06563bf1a22ab3/pkg/minikube/registry/drvs/podman/podman.go#L90-L103

**Comment by afbjorklund** [comment]
I suppose this should be made to parse eventually, workaround meanwhile is to use a released version (like 2.0.0)

**Comment by afbjorklund** [comment]
Actually it was fine with the "-dev", it was the "\n" that was the problem here
